docs/extensions/my-first-extension/_posts/1970-01-04-CreatingShortcutAndScreen.md

```diff
@@ -6,27 +6,20 @@ section: My first extension
 ---
 
 # Creating a Screen and Shortcut
-<hr />
 
-Extensions can have multiple screens in the app. Screens are [React components](https://facebook.github.io/react/docs/react-component.html) that represent a mobile screen. We want the extension to have 2 screens: one for the list of the restaurants and another for the details of each particular restaurant the user taps.
+Extensions can have multiple screens in the app. Screens are [React components](https://facebook.github.io/react/docs/react-component.html) that represent a mobile screen. We want our Restaurants extension to have 2 screens; one for the list of the restaurants (which we already made in [Getting Started]({{ site.url }}/docs/extensions/tutorials/getting-started)) and another for the details of each particular restaurant the user taps.
 
-Since the app needs to know which screen to open the first for some extension, we need to create a ***shortcut*** when creating that screen. A shortcut is a link to the starting screen of an extension. It's the item in the app navigation which opens the starting screen when a user taps on it.
+Since the app needs to know which screen to open first for some extension, we need to create a ***shortcut*** when creating that screen. A shortcut is a link to the starting screen of an extension. It's the item in the Main Navigation which opens the starting screen when a user taps on it.
 
-The list of restaurants is going to be the first screen, so let's create it and give it a shortcut:
+When we created the List screen with a shortcut in [Getting Started]({{ site.url }}/docs/extensions/tutorials/getting-started) using:
```

Comment:
Without "When". Just say "We already created ...". This is maybe too long and complex sentence.